toolchain/cc_toolchain_config.bzl

```
@@ -340,7 +345,7 @@ def cc_toolchain_config(
         dbg_compile_flags = dbg_compile_flags,
         opt_compile_flags = opt_compile_flags,
         cxx_flags = cxx_flags,
-        link_flags = link_flags,
+        link_flags = link_flags + select({"//conditions:default": [], "@@toolchains_llvm~//toolchain/config:use_libunwind": libunwind_link_flags}),
```

Canonical label literals (those with `@@`) should only be used in generated code as they can change between Bazel versions. Can you try this instead? 


```suggestion
        link_flags = link_flags + select({"//conditions:default": [], str(Label("@toolchains_llvm//toolchain/config:use_libunwind")): libunwind_link_flags}),
```

---

done